.coderabbit.yaml

```diff
@@ -13,7 +13,7 @@ reviews:
     path_instructions:
         - path: "**/*.js"
           instructions: >-
-            - Review the JavaScript code against the Google JavaScript style guide and point out any mismatches
+            - Review the JavaScript code against the [Google JavaScript style guide](https://google.github.io/styleguide/jsguide.html) and point out any mismatches
```

Comment:
There's a note on the site saying

> Please note: This guide is no longer being updated. Google recommends migrating to TypeScript, and following the [TypeScript guide](https://google.github.io/styleguide/tsguide.html).

Do we rather want to include that?

Reply:
We do not use TypeScript, so that doesn't seem right. I think this is fine for now. We may want to investigate other options for JS guides that are maintained.

Reply:
Figured as much. The "Google JavaScript style guide" I got from examples of what other people online use and I figured that it sounds OK, but definitely open to investigate this further.